google/cloud/storage/internal/rest/stub.cc

```diff
@@ -165,7 +179,7 @@ StatusOr<ListBucketsResponse> RestStub::ListBuckets(
     ListBucketsRequest const& request) {
   RestRequestBuilder builder(
       absl::StrCat("storage/", options.get<TargetApiVersionOption>(), "/b"));
-  auto auth = AddAuthorizationHeader(options, builder);
+  auto auth = AddHeaders(options, builder);
```

Comment:
I would suggest changing the name of this variable as it is no longer just the auth header.